docs/aws-parameter-store.md

```diff
@@ -56,6 +56,15 @@ In this mode, if a risk was previously found in Vault, the scan results will rep
 vault-radar scan-aws-parameter-store -r <REGION CODE> -o <PATH TO OUTPUT>.csv --index-file <PATH TO VAULT INDEX>.jsonl
 ```
 
+### HCP connection scanning behavior
+
+The default behavior of scan commands that have `hcp-upload` functionality is to require an HCP cloud connection to scan by default. This is to ensure that hashes are generated using a shared salt from the cloud keeping consistency across scans. In order to populate the HCP connection information needed you view the docs [here](hcp-upload.md).
```

Comment:
This is the default behaviour even for the scan commands that don't have HCP upload functionality

Reply:
Ah yes that is true, and since this was a copy/pasta it's everywhere. I'll update the language.